api.md

```diff
@@ -469,7 +535,66 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
 
+Detach a data node from a hypertable or from all hypertables.
+
+This function is typically used if a data node should no longer be
+used. Then it need to be removed from all hypertables that use
+it. Another reason for detaching a data node from a hyper table is
+that you want to have fewer
```

Comment:
Hmm - looks like a commit might not have been pushed?

Reply:
Fixed.